### PR TITLE
#326: [eslint-config] update buildo/eslint-config (closes #326)

### LINKED ISSUE #326: [eslint-config] update buildo/eslint-config
## requirements
update `buildo/eslint-config` and relative dependencies (`eslint`, ...) to the latest release

### PR DESCRIPTION
**Pull request by FrancescoCioria**
Issue #326

### CODE REVIEW

File: src/table/Table.js [hunk 1]
```diff
@@ -18,18 +18,18 @@ export const checkPropsInvariants = (props) => {
...
     toReturn = false;
   } else if (!multipleSelectionEnabled && onRowsSelect) {
-    warn(`'multipleSelectionEnabled' is false, so 'onRowsSelect' will never be called. Use 'onRowSelect' (singular) instead.`);
+    warn('\'multipleSelectionEnabled\' is false, so \'onRowsSelect\' will never be called. Use \'onRowSelect\' (singular) instead.');
     toReturn = false;
   }
 
   if ((!width && !height) && !autoSize) {
-    warn(`when 'autoSize=false' you must pass 'width' and 'height' otherwise the table will be sizeless.`);
+    warn('when \'autoSize=false\' you must pass \'width\' and \'height\' otherwise the table will be sizeless.');
```
Comment (giogonzo) [inline src/table/Table.js]: why was this needed?

Comment (FrancescoCioria) [inline src/table/Table.js]: mmm no idea, it was automatically done with `npm run lint-fix`

Comment (giogonzo) [inline src/table/Table.js]: ok weird

Comment (FrancescoCioria) [inline src/table/Table.js]: looks like `quotes` has become stricter
